main.go

```diff
@@ -22,11 +26,21 @@ func apiHandler(w http.ResponseWriter, r *http.Request) {
 	version := r.FormValue("version")
 	eventID := r.FormValue("eventID")
 	dateTime := r.FormValue("dateTime")
-	fmt.Printf("apikey %s, product %s, version %s, uuid %s, eventID %s, dateTime %s, path %s\n",
+	fmt.Fprintf(b, "apikey %s, product %s, version %s, uuid %s, eventID %s, dateTime %s, path %s\n",
 		apikey, product, version, uuid, eventID, dateTime, path)
 }
 
 func main() {
+	// creates a new log file with a timestamped name
+	t := time.Now()
+	fname := "ingalog_" + t.Format("20060102150405") + ".log"
```

Comment:
If we use `.log` as an extension, we don't need `log` in the first part.
So, replace `ingalog_` with `inga_`.

Reply:
yup.